FreeRTOS-Plus/Source/Application-Protocols/ReadMe_coreMQTT.url

```diff
@@ -2,4 +2,4 @@
 Prop3=19,11
 [InternetShortcut]
 IDList=
-URL=https://www.freertos.org/mqtt_lts/
+URL=https://www.freertos.org/coremqtt
```

Comment:
Shouldn't this be: `https://www.FreeRTOS.org/coremqtt/index.html`?

Reply:
Repeat of previous.